bbp/modules/default.nix

```diff
@@ -1660,6 +1660,14 @@ let
         };
 
 
+        tensorflow = pkgs.envModuleGen rec {
+            name = "tensorflow";
+            setRoot = "TENSORFLOW";
+            description = "caffe  module generated by nix";
+            packages = [ pkgs.tensorflow ];
```

Comment:
caffe module -> tensorflow  module

Reply:
Thx Pramod, I fixed that in the followed updated pull request on tf dependencies